account_cutoff_base/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Account Cutoff Base
+===================
+
+This module contains objets, fields and menu entries that are used by
+other cut-off modules ; it doesn't provide useful features by itself. You
+need to install other cut-off modules to get the useful features:
+
+* the module *account_cutoff_prepaid* will manage prepaid cut-offs based on
+  start date and end date,
+
+* the module *account_cutoff_accrual_picking* will manage the accruals based
+  on the status of the pickings.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/89/10.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/account-closing/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Alexis de Lattre <alexis.delattre@akretion.com>
```

Comment:
judging from the commits this list should be updated w/ some ppl from ACSONE and C2C